torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -268,21 +272,22 @@ xla::OpSharding ShardingUtil::CreateOpSharding(
       sharding = xla::HloSharding::PartialTile(new_tile_assignment).ToProto();
       break;
     }
+    case ShardingType::UNKNOWN: {
+      sharding = xla::HloSharding::Unknown().ToProto();
+    }
     default: {
       TF_LOG(ERROR) << "Invalid arguments: sharding_type " << sharding_type;
     }
   }
-  TF_VLOG(INFO) << "OpSharding (ShardingType: " << sharding_type << "):\n"
-                << sharding.DebugString()
-                << ", sharding.type()=" << sharding.type();
   return sharding;
```

Comment:
why removed?

Reply:
Too verbose, as it prints per tensor. I found it more annoying than useful.